src/styles/blocks/button.scss

```diff
@@ -0,0 +1,41 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: 100%;
+  height: 56px;
+
+  text-align: center;
+
+  border: 2px solid $colorBlue;
+  border-radius: 8px;
+
+  transition: all 0.3s;
+
+  cursor: pointer;
+
+  &--primary {
+    background-color: $colorBlue;
+    border-color: $colorBlue;
+    box-shadow: 0 6px 23px rgba(0, 117, 255, 0.46);
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 14px;
+    color: #fff;
```

Comment:
you can create variable for white color too, but I don't know if that is necessary 🦧